README.md

```diff
@@ -1,2 +1,34 @@
-# webpack-demo
- 
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# LeaderBoard with API
+
+Working with APIs
+
+## Built with
+
+- Vanilla JavaScript
+- ESLint
+- WebPack
+
+## Author
+
+**👤 Kenny Ortega**
```

Comment:
- [ ] Kindly let's provide instructions on how to set up and run the project locally in the README, to help other people easily figure out how to do it and thus keep the repo more professional 😉